### PR TITLE
Fix nxp 19031 resources cache

### PR DESCRIPTION
**Pull request by atchertchian**
See https://jira.nuxeo.com/browse/NXP-19031 for main task (remaining commit is just cleanup): this is about generating a timestamp to include in the resources URL to handle proper reload of cache on the browser-side in case of changes to resources.

Changes for NXP-19031 (so not including the cleanup) should be backported until 7.10.

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

File: nuxeo-jsf/nuxeo-web-resources-jsf/src/main/java/org/nuxeo/ecm/web/resources/jsf/AbstractResourceRenderer.java [hunk 1]
```diff
@@ -81,6 +83,23 @@ protected String resolveUrl(FacesContext context, UIComponent component) throws
...
         }
+        return resolveUrlWithTimestamp(component, url);
+    }
+
+    protected String resolveUrlWithTimestamp(UIComponent component, String url) {
+        boolean doIncludeTimestamp = true;
+        Object includeTimestamp = component.getAttributes().get("includeTimestamp");
+        if (includeTimestamp instanceof String) {
+            if (!StringUtils.isBlank((String) includeTimestamp)) {
+                doIncludeTimestamp = Boolean.valueOf((String) includeTimestamp);
```
Comment (efge) [inline nuxeo-jsf/nuxeo-web-resources-jsf/src/main/java/org/nuxeo/ecm/web/resources/jsf/AbstractResourceRenderer.java]: `Boolean.parseBoolean`

**Comment by efge** [comment]
👍

**Comment by atchertchian** [comment]
thanks, will let test&push merge

**Pull request closed without merging**